prepare.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# install libjpeg-dev with apt
+sudo apt-get install libjpeg-dev
```

Comment:
Das sollte in der `prepare_ubuntu.sh` liegen, oder? Manche haben nicht das Problem, daher würde ich die Zeile auskommentieren und die README anpassen.